# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/tests/unit/test_sandbox_DOMException.js
# Commit: 942a92874526
# Full Hash: 942a92874526e7a18167af89e358d69cbaeab268
# Author: Boris Zbarsky <bzbarsky@mit.edu>
# Date: 2020-02-22 09:52:08
# Regressor Bug: 267645
# File Overlap Count: 1
# Description:
#   Bug 267645 part 1.  Add a way to ask for DOMException to be exposed in sandboxes. r=bholley
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D63278
# ==============================================================================

diff -r d21671811dff -r 942a92874526 js/xpconnect/tests/unit/test_sandbox_DOMException.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/xpconnect/tests/unit/test_sandbox_DOMException.js	Sat Feb 22 04:56:52 2020 +0000
@@ -0,0 +1,10 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+function run_test() {
+  var Cu = Components.utils;
+  var sb = new Cu.Sandbox('http://www.example.com',
+                          { wantGlobalProperties: ["DOMException"] });
+  sb.notEqual = Assert.notEqual.bind(Assert);
+  Cu.evalInSandbox('notEqual(DOMException, undefined);', sb);
+}